### DISCUSSION TITLE
Locations / Contacts

### DISCUSSION THREAD

**Discussion by na-nederland** [code4recovery/12-step-meeting-list#1242] [General]
We have started using the plugin recently for the new NA website in the Netherlands, and we are **very grateful** for being able to use it!  https://www.nanederland.nl/meetings/ We are using Legacy UI at this time and have the TSML Xtras installed.

First: I'm not the webmaster but the fairly technical operations guy :)

The questions we have:
Locations: Since locations are "entities" that can be reused, we would expect a menu option that would allow editing of locations.  
Contacts (group): same thing. 

I'm not saying that it's not working, just wondering if we are missing a setting /extra plugin.

In service, Ewout

**Comment by anchovie91471** [comment]
Hi Ewout! We're glad you're finding the plugin useful!

Yes, TSML doesn't include the Groups/Locations as separate screens in the menu, even though they are technically WP entities. They can, however, be edited on a related meeting edit screen, and, being that they are entities, they are changed for every associated meeting.

This is an experimental feature in TSML Xtras that isn't available yet, but may be on the roadmap.

**Reply by na-nederland** [reply]
Thanks for the explanation, much appreciated!

**Reply by na-nederland** [reply]
The thing is that locations without meetings remain (searchable and indexed). Meaning that we have multiple locations without meetings. 

Is there a way to remove locations without meetings?
